daprdocs/content/en/reference/components-reference/supported-secret-stores/azure-keyvault.md

```diff
@@ -47,7 +47,7 @@ The above example uses secrets as plain strings. It is recommended to use a loca
 
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
-| vaultName          | Y        | The name of the Azure Key Vault | `"mykeyvault"`
+| vaultName          | Y        | The name of the Azure Key Vault, if you only provide name, it would covert to "[your_keyvault_name].vault.azure.net" in Dapr. If your URL uses other suffix, please provide whole link like "test.vault.azure.cn". we support all suffixes that Azure uses. | `"mykeyvault"`, `mykeyvault.value.azure.cn`
```

Comment:
Is this new behavior that's being introduced? Looking at the current implementation, this doesn't appear to line up https://github.com/dapr/components-contrib/blob/master/secretstores/azure/keyvault/keyvault.go#L134

Reply:
Yep, the pr is https://github.com/dapr/components-contrib/pull/944

Reply:
```suggestion
| vaultName          | Y        | The name of the Azure Key Vault. If you only provide A name, it would covert to `[your_keyvault_name].vault.azure.net` in Dapr. If your URL uses another suffix, please provide whole link, like `test.vault.azure.cn`.  | `"mykeyvault"`, `"mykeyvault.value.azure.cn"`
```

Reply:
```suggestion
| vaultName          | Y        | The name of the Azure Key Vault. If you only provide a name, it will covert to `[your_keyvault_name].vault.azure.net` in Dapr. If your URL uses another suffix, please provide the entire URI, such as `test.vault.azure.cn`. | `"mykeyvault"`, `"mykeyvault.value.azure.cn"`
```

Reply:
```suggestion
| vaultName | Y | The name of the Azure Key Vault. If you only provide a name, it will covert to `[your_keyvault_name].vault.azure.net` in Dapr. If your URL uses another suffix, please provide the entire URI, such as `test.vault.azure.cn`. | `"mykeyvault"`, `"mykeyvault.value.azure.cn"`
```